sdk/core/azure-core/inc/azure/core/http/curl/curl connection_pool.hpp

```diff
@@ -26,83 +26,161 @@ namespace Azure { namespace Core { namespace Test {
 #endif
 
 namespace Azure { namespace Core { namespace Http {
+  /**
+   * @brief The available options to set libcurl ssl options.
+   *
+   * @remark The SDK will map the enum option to libcurl's specific option. See more info here:
+   * https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
+   *
+   */
+  struct CurlTransportSSLOptions
+  {
+    bool AllowBeast = false;
+    bool NoRevoke = false;
+    bool NoPartialchain = false;
+    bool RevokeBestEffort = false;
+    bool NativeCa = false;
+  };
 
   /**
-   * @brief CURL HTTP connection pool makes it possible to re-use one curl connection to perform
-   * more than one request. Use this component when connections are not re-used by default.
+   * @brief Set the curl connection options like a proxy and CA path.
    *
-   * This pool offers static methods and it is allocated statically. There can be only one
-   * connection pool per application.
    */
-  class CurlConnectionPool {
-#ifdef TESTING_BUILD
-    // Give access to private to this tests class
-    friend class Azure::Core::Test::TransportAdapter_connectionPoolTest_Test;
-#endif
-  public:
+  struct CurlTransportOptions
+  {
     /**
-     * @brief Mutex for accessing connection pool for thread-safe reading and writing.
+     * @brief The string for the proxy is used without any parsing to libcurl handle.
```

Comment:
`is **used** ... to libcurl handle` doesn't read well.
```suggestion
     * @brief The string for the proxy is passed directly to the libcurl handle without any parsing
```